scripts/create-user.sh

```diff
@@ -227,7 +196,12 @@ get_subject() {
 
 # Expand variables from message body, escaping new lines and quotes
 get_body() {
-  eval "echo \"${MESSAGE}\"" | \
+  erb email="${EMAIL}" org="${ORG:-}" invite_url="${INVITE_URL}" "${SCRIPT_DIR}/templates/create_user_plain.erb" | \
```

Comment:
Could you add something to the `check_params_and_environment` function to ensure that `erb` is available early on? Otherwise the account could be created/reset but the email would later fail to render and send.

Reply:
Yes, good idea. Added.